hw6_functions/homework6_task3.py

```diff
@@ -0,0 +1,16 @@
+"""
+    Task3.Write a function is_prime that takes 1 argument - a number from 2 to 1000, and returns
+    True if it is a prime number, and False otherwise.
+"""
+
+
+def is_prime(number: int) -> bool:
+    if number in range(2, 1001):
+        for _ in range(2, number):
```

Comment:
if you use this var in the loop you should give it a name. 
_ only if you iterate without using a variable

Reply:
updated according to your notice